contribs/gnodev/pkg/dev/node.go

```diff
@@ -92,6 +93,9 @@ type Node struct {
 var DefaultFee = std.NewFee(50000, std.MustParseCoin(ugnot.ValueString(1000000)))
 
 func NewDevNode(ctx context.Context, cfg *NodeConfig) (*Node, error) {
+	stdlibsDeployer := crypto.MustAddressFromString("g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5") // test1, FIXME: replace
```

Comment:
This is fixable I think with @Villaquiranm's PR

Reply:
Yeah I think I'll wait on it